Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/RestrictedActionsApi/RestrictedActionsAPIWrapper.cs

```diff
@@ -24,21 +24,21 @@ public void Dispose()
 
         [UsedImplicitly]
         public void MovePlayerTo(
-            int newRelativePositionX, int newRelativePositionY, int newRelativePositionZ)
+            double newRelativePositionX, double newRelativePositionY, double newRelativePositionZ)
         {
             api.TryMovePlayerTo(
-                new Vector3(newRelativePositionX, newRelativePositionY, newRelativePositionZ),
+                new Vector3((float)newRelativePositionX, (float)newRelativePositionY, (float)newRelativePositionZ),
                 null);
         }
 
         [UsedImplicitly]
         public void MovePlayerTo(
-            int newRelativePositionX, int newRelativePositionY, int newRelativePositionZ,
-            int cameraTargetX, int cameraTargetY, int cameraTargetZ)
+            double newRelativePositionX, double newRelativePositionY, double newRelativePositionZ,
```

Comment:
Leaving two questions!
1) Is it possible to put float here to avoid the casting below? 
2) If js call this method with ints, wouldnt you have the same issue?

Reply:
no, ClearScript marshals floating point numbers as `double`